packages/core/test/unit/query-generator/remove-constraint-query.test.ts

```diff
@@ -27,6 +27,9 @@ describe('QueryGenerator#removeConstraintQuery', () => {
           dialect.name,
           ['ifExists'],
         ),
+        oracle: buildInvalidOptionReceivedError('removeConstraintQuery', dialect.name, [
```

Comment:
This should become the default and what is now the default should become 'postgres mariadb mssql'

Reply:
Done.